plugins/woocommerce/changelog/update-53563-handle-beta-testing-plugin-installs

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: enhancement
+Comment: Take a best-effort approach when matching payment gateway installed or active plugins with payment extension suggestions and reduce false-negatives.
```

Comment:
Because the wp/data is developer focused and folks should know about the fix, while the Payment NOX fix is internal at this point and I don't want to fill up the WooCommerce changelog.

Reply:
That makes sense!